backend/shared/src/websockets/helpers.ts

```diff
@@ -90,8 +90,12 @@ export function broadcastNewAnswer(answer: Answer) {
 }
 
 export function broadcastUpdatedAnswer(answer: Answer) {
-  const payload = { answer }
-  const topics = [`contract/${answer.contractId}/updated-answer`]
+  broadcastUpdatedAnswers([answer])
```

Comment:
We want old clients to still receive websocket updates on the old topic, so this should not just call the new function.